### PR TITLE
Sd 83

### PR DESCRIPTION
**Pull request by asnewman**
Individual stock page.
Does not include transaction modal.

### CODE REVIEW

File: Web/stockdogweb/src/components/Trade/Trade.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import './Trade.css';
+import Button from '../Button/Button';
+
+class Trade extends Component {
+   render() {
+      return (
+         <div className='Trade'>
+            <div className='trade-elements'>
+               <div className='trade-element'>
+                  <h1>13</h1>
```
Comment (sshaul) [inline Web/stockdogweb/src/components/Trade/Trade.js]: Are these numbers going to come from props?

File: Web/stockdogweb/src/containers/Stock/Stock.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import Trade from '../../components/Trade/Trade';
+
+class Stock extends Component {
+   render() {
+      return (
+         <div className="Stock">
+            <Navbar />
+            <Graph />
+            <div className="stock-content">
+               <Article title={"About"} content={'Advanced Micro Devices, Inc. engages in the provision of semiconductor businesses. It operates through the Computing and Graphics and Enterprise, Embedded and Semi-Custom segments.'}/>
```
Comment (sshaul) [inline Web/stockdogweb/src/containers/Stock/Stock.js]: Where are you going to get this content from too?

Comment (asnewman) [reply Web/stockdogweb/src/containers/Stock/Stock.js]: API call

Comment (asnewman) [reply Web/stockdogweb/src/containers/Stock/Stock.js]: State. Lemme change it

File: Web/stockdogweb/src/components/Trade/Trade.js (hunk 1 above)
Comment (asnewman) [reply Web/stockdogweb/src/components/Trade/Trade.js]: Yes let me change it

**Pull request merged**